src/components/MedCentersPage/MedCentersList/MedCenter/CenterStaff/CenterStaff.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-
+import uniqid from 'uniqid';
```

Comment:
do we really need this lib on the frontend?

Reply:
removed

Reply:
@KLIM-nk01 not removed